### DISCUSSION TITLE
How to change default settings?

### DISCUSSION THREAD

**Discussion by silver-ac** [gildas-lormeau/SingleFile#962] [General]
Well, I want to change the default setting to auto save all tabs. It there anything I can do with the code?

**Comment by gildas-lormeau** [comment]
You have to run this code in the background script:

```js
const tabsData = (await browser.storage.local.get()).tabsData;
tabsData.autoSaveAll = true
await browser.storage.local.set({ tabsData });
```

**Comment by gildas-lormeau** [comment]
I'm moving this issue in the Discussions tab.
